lib/text_parser.dart

```diff
@@ -958,6 +963,97 @@ class TextParser extends StatelessWidget {
       child: widget,
     );
   }
+
+  Widget _addWidgetSpanToWidget({required Widget widgetSpan, required Widget nodeParsed}) {
+    if (nodeParsed is CleanRichText) {
+      return _addWidgetSpanToCleanRichText(widgetSpan: widgetSpan, richText: nodeParsed);
+    } if (nodeParsed is Column && nodeParsed.children.isNotEmpty) {
+      return _addWidgetSpanToColumn(widgetSpan: widgetSpan, column: nodeParsed);
+    } else {
+      return CleanRichText(
+        _optimizeTextspan(TextSpan(
```

Comment:
the call to `_optimizeTextSpan` isn't needed here as it had already been optimized previously

Reply:
I do not think so. Because I passed 1 more InlineSpan to it. So this InlineSpan is not optimized yet